pkgs/development/julia-modules/depot.nix

```diff
@@ -52,7 +52,10 @@ runCommand "julia-depot" {
   # for finding the extra packages we need to add
   python ${./python}/find_package_implications.py "${closureYaml}" '${lib.generators.toJSON {} packageImplications}' extra_package_names.txt
 
-  # git config --global --add safe.directory '/nix'
+  # Work around new git security features added in git 2.44.1
+  # See https://github.com/NixOS/nixpkgs/issues/315890
+  git config --global --add safe.directory '*'
```

Comment:
Any way we can simply restrict this to dirs in `/nix/store`?

Reply:
> Nix can be used with custom store paths.

`builtins.storeDir`, then.

Reply:
It doesn't appear that the `safe.directory` option supports parent directories or wildcards outside of the single `*`. I tried the following strings:

```
'${config.nix.storeDir or builtins.storeDir}'
'${config.nix.storeDir or builtins.storeDir}/*'
'${config.nix.storeDir or builtins.storeDir}/**/*'
```

Reply:
Ah, [`*` is not a wildcard](https://git-scm.com/docs/git-config/2.45.0#Documentation/git-config.txt-safedirectory).